src/components/Network.js

```diff
@@ -103,15 +132,12 @@ class Network extends React.Component {
     };
 
     const renderConnected = (peer, conn) => {
-        const sync = conn.plugins["node-sync-timestamp"];
-        const first_timestamp = this.state.dag.first_timestamp;
-        const latest_timestamp = this.state.dag.latest_timestamp;
-        const delta = latest_timestamp - first_timestamp;
-        const synced_percent = 100 * (sync.synced_timestamp - first_timestamp) / delta;
-        let general_percent = (100 * (sync.latest_timestamp - first_timestamp) / delta) - synced_percent;
-        if (general_percent > 100) {
-          general_percent = 100;
-        }
+        const sync_data = {
```

Comment:
What fields should we expect in this object? Add them and their types in the comments, please.

Reply:
Done, thanks.